controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -236,15 +239,45 @@ void wfa_ca::handle_wfa_ca_message(
 
     switch (command_type) {
     case eWfaCaCommand::CA_GET_VERSION: {
-        // TODO
+
+        /* 
```

Comment:
This and many of the following lines had a spurious space at the end of the line. Please configure your editor to avoid that.

End-of-line space is bad because it can easily lead to spurious differences in later edits.

I've fixed this up while rebasing.

Reply:
I don't have such a setting in my editor